### ISSUE TITLE
Duplicate interface definition for class 'RCTView'

### ISSUE THREAD

**Issue by reason211** [shoutem/ui#134]
I follow the following steps to do step by step, but it can not be compiled successfully.

```
$ react-native init HelloWorld && cd HelloWorld

$ npm install --save @shoutem/ui

$ react-native link

$ react-native run-ios
```

![2017-01-19 10 46 30-min](https://cloud.githubusercontent.com/assets/6103174/22111753/6b3c8f02-de9b-11e6-886b-4faa35a4f474.png)

Build Environment:
Xcode: Version 8.2.1 (8C1002)
react-native": "0.40.0"
node: v6.3.1
npm: 3.10.3

Is it because shoutem-ui does not support react-native 0.40.0 yet ?

**Comment by tedjames** [comment]
I'm having issues with BVLinearGradient as well. RN 0.40 does introduce some breaking changes so it's likely that the shoutem team has not yet released a version that supports RN 0.40. 

Can someone please confirm whether or not Shoutem UI and Animations support RN 0.40 and if not, when should we expect a new release?

**Comment by tedjames** [comment]
_**Solution Found**_
1. Open your project in xcode and navigate to libraries/BVLinearGradient.xcodeproj/BVLinearGradient

2. In each of the four files within that directory, you need to change a few import statements at the top of the file. Look for input statements with a prefix of RCT like:
```
#import "RCTBridge.h"
#import "RCTViewManager.h"
#import "RCTView.h"
```
3. Now change each of those statements to this:
```
#import <React/RCTBridge.h>
#import <React/RCTViewManager.h>
#import <React/RCTView.h>
```

Moving forward you may discover other libraries giving you issues like this. Until these libraries are updated to work with the latest 0.40 release of react native, you'll have to continue with workarounds like this. 

Simply put, you just need to reformat import statements prefixed with RCT from being enclosed in "{library name}" to being enclosed within <React/ {library name} >
 
This fix should work for most other issues related to these breaking changes that came with 0.40.

**Comment by reason211** [comment]
@tedjames It's work! Your help was greatly appreciated!

**Comment by empitechathu** [comment]
@tedjames You are a life saver :D thanks mate

**Comment by benjick** [comment]
Is this in 0.10.9? Getting similar errors in RN 0.41.2

**Comment by PoloGarcia** [comment]
Getting the same errors in 0.41.2 too :( any idea when this is going to get pushed?

**Comment by dekkofilms** [comment]
Also, if you get a similar error in relation to the vector icons library. Thanks to the solution above, find the RNVectorIcons.xcodeproj/RNVectorIconsManager/RNVectorIconsManager.h, and replace:

`#import "RCTLog.h"`

with

`#import <React/RCTLog.h>`

Thanks, @tedjames

**Comment by ryanhomer** [comment]
Note that this is fixed in more recent versions of react-native-linear-gradient. In particular, version 2.0.0 solved it for me.

**Comment by SoHotSoup** [comment]
Guys, there is 0.12.0-rc.1 available at npm with dist-tag rc so could you
try it and verify does it fix this issue?

**Comment by Rubanlycan** [comment]
> #import "RCTBridge.h"
> #import "RCTViewManager.h"
> #import "RCTView.h"

@tedjames my files are already have statements like you mentioned but then also i m getting error
